source/graphql/traits.d

```diff
@@ -467,3 +457,92 @@ template PossibleTypesImpl(Type, AllTypes...) {
 		}
 	}
 }
+
+// compiler has a hard time inferring safe. So we have to tag it.
+@safe
+void execForAllTypes(T, alias fn, Context...)(auto ref Context context)
+{
+	// establish a seen array to ensure no infinite recursion.
+	execForAllTypesImpl!(T, fn)((bool[void*]).init, context);
+}
+
+@trusted private void *keyFor(TypeInfo ti)
+{
+	return cast(void*)ti;
+}
+
+@safe
+void execForAllTypesImpl(Type, alias fn, Context...)(
+									 bool[void *] seen, auto ref Context context)
+{
```

Comment:
"void *" -> "void*"

Reply:
hehe, took me a minute to see what you are talking about because of the markdown.